__fixtures__/nodeReleases.tsx

```diff
@@ -0,0 +1,97 @@
+import type { NodeRelease } from '../types';
+
+export const createNodeReleases = (): NodeRelease[] => [
```

Comment:
I've updated the fixture as well.
Let's not use `as NodeRelease[]`, it doesn't throw an error when it should.